indigo_app/views/works.py

```diff
@@ -56,6 +56,29 @@ def determine_place(self):
     def get_context_data(self, **kwargs):
         context = super(WorkViewBase, self).get_context_data(work=self.work, **kwargs)
         context['work_json'] = json.dumps(WorkSerializer(instance=self.work, context={'request': self.request}).data)
+
+        # add other dates to timeline
+        work_timeline = self.work.points_in_time()
+        other_dates = [
+            ('assent_date', self.work.assent_date),
+            ('commencement_date', self.work.commencement_date),
+            ('repealed_date', self.work.repealed_date)
+        ]
+        # add to existing dates (e.g. if publication and commencement dates are the same)
+        for date in work_timeline:
+            for name, other_date in other_dates:
+                if date['date'] == other_date:
+                    date['{other_date}'.format(other_date=other_date)] = True
```

Comment:
This seems weird. What are you trying to do?

`str(other_date)` may be simpler than a format string, if that's what you want. Why do you want a date string to be True?

Reply:
I think you want `date[name] = True`L